src/test/shell/bazel/remote/remote_execution_test.sh

```diff
@@ -3286,8 +3250,7 @@ EOF
   bazel build \
     --remote_cache=grpc://localhost:${worker_port} \
     --remote_download_toplevel \
-    --experimental_action_cache_store_output_metadata \
-    --verbose_failures \
+    $extra_build_flags \
```

Comment:
I suspect bash linter may not like that -- your best bet may be to use the "$@" array directly or declare and use extra_build_flags as an array and use like that (personally, I recommend `"$@"`):

```
local -r extra_build_flags=("$@")
...
"${extra_build_flags[@]}"
```